src/main/java/net/william278/velocitab/tab/PlayerTabList.java

```diff
@@ -214,6 +214,10 @@ public CompletableFuture<Component> getFooter(@NotNull TabPlayer player) {
 
     // Update the tab list periodically
     private void updatePeriodically(int updateRate) {
+        // Set the minimal update rate to 200 milliseconds
+        if (updateRate < 200) {
+            updateRate = 200;
+        }
```

Comment:
```java
        updateTask = plugin.getServer().getScheduler()
                .buildTask(plugin, () -> {
                    if (players.isEmpty()) {
                        return;
                    }
                    players.forEach(player -> {
                        this.updatePlayer(player);
                        player.sendHeaderAndFooter(this);
                    });
                })
                .repeat(Math.max(200, updateRate), TimeUnit.MILLISECONDS)
                .schedule();
```
I think instead of this, I'd rather see `Math#max` applied -- just a little cleaner :)